tests/test_core.py

```diff
@@ -24,48 +26,60 @@ def setUp(self):
         self.gps_inf = -1000.0
 
     def tearDown(self):
+        time.sleep(20)
         os.close(self.db_fd)
         os.unlink(self.db_name)
 
     def test_0a_disable(self):
+        time.sleep(20)
         code = core.disable(self.iface)
         self.assertEqual(code, 0)
 
     def test_0b_interfaces(self):
+        time.sleep(20)
         ifaces = core.interfaces()
         self.assertIsInstance(ifaces, list)
         self.assertNotIn(self.iface, ifaces)
 
     def test_0c_status(self):
+        time.sleep(20)
         status = core.status(self.iface)
         self.assertEqual(status, '')
 
     def test_0d_scan(self):
+        time.sleep(20)
         self.assertRaises(core.WifiException, core.cell_all, self.iface)
 
     def test_0e_available(self):
+        time.sleep(20)
         self.assertRaises(core.WifiException, core.available, self.iface)
 
     def test_0f_save(self):
+        time.sleep(20)
         self.assertRaises(core.WifiException, core.save, self.iface, 'foo', 'foo', self.db)
 
     def test_0g_connect(self):
+        time.sleep(20)
         self.assertRaises(core.WifiException, core.connect, self.iface, 'foo', 'foo', self.db)
 
     def test_1a_enable(self):
+        time.sleep(20)
         code = core.enable(self.iface)
         self.assertEqual(code, 0)
 
     def test_1b_interfaces(self):
+        time.sleep(20)
         ifaces = core.interfaces()
         self.assertIsInstance(ifaces, list)
-        self.assertNotIn(self.iface, ifaces)
+        self.assertIn(self.iface, ifaces)
```

Comment:
Do we have an explanation for this maybe? a comment or something would help. Same for the change in test below.